2022q4/gcc.adoc

```diff
@@ -0,0 +1,25 @@
+=== GCC: Update GCC default version to version 12
+
+Links: +
+link:https://gcc.gnu.org[GCC Project] URL: link:https://gcc.gnu.org[https://gcc.gnu.org] +
+link:https://gcc.gnu.org/gcc-11/[GCC 11 release series] URL: link:https://gcc.gnu.org/gcc-11/[https://gcc.gnu.org/gcc-11/] +
+link:https://gcc.gnu.org/gcc-12/[GCC 12 release series] URL: link:https://gcc.gnu.org/gcc-12/[https://gcc.gnu.org/gcc-12/]
+
+Contact: Lorenzo Salvadore <salvadore@FreeBSD.org> +
+
+ * The GCC default version has now been updated to version 12.
+   Thank you very much to antoine@ for running the necessary exp-runs and to all the contributors, maintainers and committers involved in the process.
+   As it has been noted last quarter, for every major version of GCC FreeBSD usually waits for the release of the second minor version to update GCC default version. However next time I would like to attempt to update the default version as soon as the first minor version of GCC 13 is out. The rationale for waiting the second minor release is to wait for other operating systems (in particular Linux distributions) to find the bugs, report them, and fix them, so that FreeBSD could focus mainly on FreeBSD specific cases. But this also means that upstream software only hear from FreeBSD rarely and mostly when it concerns FreeBSD only, thus our operating system risks to be considered as minor and unimportant for them.
```

Comment:
```suggestion
   As it has been noted last quarter, for every major version of GCC FreeBSD usually waits for the release of the second minor version to update GCC default version. However next time I would like to attempt to update the default version as soon as the first minor version of GCC 13 is out. The rationale for waiting the second minor release is to wait for other operating systems (in particular Linux distributions) to find the bugs, report them, and fix them, so that FreeBSD could focus mainly on FreeBSD specific cases. But this also means that upstream software developers only hear from FreeBSD rarely and mostly when it concerns FreeBSD only, thus our operating system risks being considered minor and unimportant for them.
```